### ISSUE TITLE
Translate all comments into English (if possible)

### ISSUE THREAD

**Issue by TommiPrami** [LUXOPHIA/LUX#1]
Consider English translation.

Now that ?Japanese? makes bit hard to follow the code.

**Comment by luxidea** [comment]
Sorry.
【クラス】→【CLASS】
【レコード】→【RECORD】
Is that part?
There are no important comments.
Rather, you need to create a document, which is difficult for me.

**Comment by TommiPrami** [comment]
OK, if can use English it's good for community, but definitely not must if code is clean
